controllers/argocd/deployment.go

```diff
@@ -831,7 +841,24 @@ func (r *ReconcileArgoCD) reconcileRepoDeployment(cr *argoprojv1a1.ArgoCD) error
 		repoEnv = argoutil.EnvMerge(repoEnv, []corev1.EnvVar{{Name: "ARGOCD_EXEC_TIMEOUT", Value: fmt.Sprintf("%d", *cr.Spec.Repo.ExecTimeout)}}, true)
 	}
 
-	deploy.Spec.Template.Spec.InitContainers = getArgoRepoInitContainers(cr)
+	deploy.Spec.Template.Spec.InitContainers = []corev1.Container{{
```

Comment:
It looks like the user will no longer be able to specfic the initContainer config. It is going to be a upgrade impact. Can we continue to support initContainer without affecting sidecar container?  Is it correct to pass the `repoEnv` to the initContainers?

Reply:
The user will still be able to specify the initContainers config.  The code just below this (at line 859) appends the user-specified initContainers to the deploy.Spec.Template.Spec.InitContainers array.

I'll change the code to not pass the `repoEnv` to the copyutil init container.